_build/Extension-SetIdentityAndVersion.ps1

```diff
@@ -43,9 +43,9 @@ if($buildNumber -match $VersionRegEx){
 
   $revision =  [int]::Parse($matches[4]).ToString()
 
-  if($buildNumber.ToLower().StartsWith("dev")){
-    $revision = (65000 + [int]::Parse($matches[4])).ToString();
-  }
+  #if($buildNumber.ToLower().StartsWith("dev")){
```

Comment:
Why has this been kept but commented out? Why not just delete it?

Reply:
Your right, going to delete